neuralmonkey/runners/label_runner.py

```diff
@@ -16,9 +17,14 @@ def __init__(self,
         super(LabelRunner, self).__init__(output_series, decoder)
         self._postprocess = postprocess
 
+        # make sure the lazy decoder creates its output tensor
+        log("Decoder output tensor: {}".format(decoder.decoded))
```

Comment:
Tak si to zkus :-) Když je ten labeler jediný runner, tak e stane to, že se načítají proměnné dřív, než se postaví graf.